js-core-3/week-1/lesson.md

```diff
@@ -56,6 +72,24 @@ The purpose of this class is to introduce to the student:
 
 * https://api.example.com/v1/users?limit=10
 
+
+##### Check-in
+
+Here is an example of Spotify's REST endpoint to allow you to get the tracks from a playlist
+
+    GET https://api.spotify.com/v1/playlists/{playlist_id}/tracks
+
+**Task (5 mins):**
+
+    On Instagram, you can access a post via the following endpoint:
+
+        GET https://api.instagram.com/v1/media/{media-id}
```

Comment:
Should we remove the `GET` here to make q2 less obvious?

Reply:
Ahhh good catch! Must have accidentally left that in.